js/UrbanAirship.js

```diff
@@ -494,6 +494,31 @@ class UrbanAirship {
     return UrbanAirshipModule.isQuietTimeEnabled();
   }
 
+  /**
+  * Sets Autobadging on iOS, either to true or false. Badging is not supported for Android.
+  * 
+  * @param {boolean} enabled Whether or not to enable Autobadging.
+  */
+  static setAutobadgeEnabled(enabled: boolean) {
+    if (Platform.OS == 'ios') {
+      UrbanAirshipModule.setAutobadgeEnabled(enabled);
+    } else {
+      console.log("This feature is not supported on this platform.")
+    }
+}
+
+  /**
+   * Checks to see if Autobadging on iOS is enabled. Badging is not supported for Android.
+   * 
+   * @return {Promise.<boolean>} A promise with the result, either true or false.
+   */
+  static isAutobadgeEnabled(): Promise<boolean> {
+    if (Platform.OS != 'ios') {
```

Comment:
Could you flip this if/else to match the one in `setAutobadgeEnabled()`?

Reply:
Can do! Word of warning, I was matching the pattern that was set by setBadgeNumber and getBadgeNumber. Did you want me to change those as well in this PR?

Reply:
Nah, just do this one. It's mostly just readability.